config/rbac/role.yaml

```diff
@@ -83,6 +81,17 @@ rules:
   - list
   - update
   - watch
+- apiGroups:
+  - ceph.rook.io
+  resources:
+  - cephfilesystemsubvolumegroups
+  verbs:
+  - create
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
This seems unrelated to this PR and should not be coupled with it

Reply:
Replied here 
https://github.com/red-hat-storage/ocs-operator/pull/3070#discussion_r2063277655